jenkins/catapult/templates/UbuntuTestSanitizerBaseImage.Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM symbolplatform/symbol-server-compiler:ubuntu-clang-14
+ARG DEBIAN_FRONTEND=noninteractive
```

Comment:
1. ARG or ENV ?
2. can't you use UbuntuTestBaseImage.dockerfile?

Reply:
1. ARG is better to use since the variable is only required during the docker build.  ENV will be set when the container is running.
2. Yea, its same except I remove some tools.

Reply:
`ARG is better` you should check other usages of ARG vs ENV in dockerfiles as a post check-in task